apps/demo-game/src/reducers/PeriodReducer.ts

```diff
@@ -1,57 +1,55 @@
-import { Action } from '@gbl-uzh/platform'
+import {
+  PayloadPeriodConsolidation,
+  PayloadPeriodInitialisation,
+} from '@gbl-uzh/platform'
 import { debugLog } from '@gbl-uzh/platform/dist/lib/util'
 import { PeriodFacts, PeriodSegmentFacts } from '@graphql/index'
-import { PrismaClient } from '@prisma/client'
-import { match } from 'ts-pattern'
 
-export enum ActionTypes {
-  PERIOD_INITIALIZE = 'PERIOD_INITIALIZE',
-  PERIOD_CONSOLIDATE = 'PERIOD_CONSOLIDATE',
+type State = {
```

Comment:
we will probably pull these state types out to another file to pass them to the platform/generate validators from them as well (so we can verify with yup/zod that the data is correct before storing it to DB). to be discussed

Reply:
Good point!